### ISSUE TITLE
Consider only showcasing free software applications

### ISSUE THREAD

**Issue by lionirdeadman** [flatpak/flatpak.github.io#489]
Currently, flatpak.org shows proprietary applications when directing people to Flathub. I think it might be good to only showcase free software applications which we think are important / good instead of essentially giving marketing to proprietary applications.

Here are criterias which I think would be good to have in no particular order :

- Applications which are maintained by their developer
- Applications which are made specifically for Flatpak and/or put effort into being fully usable
- Applications which differ in toolkit used (GTK, Qt, Electron, etc.)
- Applications that are known in the Linux community

The goals of doing this would be to show support and appreciation for those who work to make things available as Flatpak, work on the Flatpak platform and to generally show a diverse view of what free software are available. I believe it would also help expel beliefs that Flatpak is made for proprietary applications and that Flatpak doesn't care about freedom.

It might be a good idea to have more than 8 applications and have them show in a random order but that may be done at a later time as it would require more work.

Here are applications that I think might be worthy of being showcased there : 

- Firefox
- Flatseal
- OBS
- GNOME Builder
- Elisa

**Comment by Seirdy** [comment]
Blender, RetroArch, and GNOME Boxes would also be good candidates.

I'm not sure it'd be a good idea to feature a QtWidgets-based app because Flatpak still doesn't support custom Qt themes outside KDE; users might wonder why their first Flatpak app looks "ugly".

**Comment by Be-ing** [comment]
> I'm not sure it'd be a good idea to feature a QtWidgets-based app because Flatpak still doesn't support custom Qt themes outside KDE; users might wonder why their first Flatpak app looks "ugly".

Is that an issue with QML too?

**Comment by lionirdeadman** [comment]
I think after further review that I'd like to not have Blender because it is not maintained by someone from the development team which is a shame. I propose LibreOffice in its place.

/me is making the PR at this moment

**Comment by Seirdy** [comment]
A game like 0 A.D. would be nice too. Ideally it'd have something in the 
"communication" category; I propose Srain 
(https://flathub.org/apps/details/im.srain.Srain).

-- 
/Seirdy

**Comment by lionirdeadman** [comment]
For a game, Veloren would be a better pick I think because of the maintainership, Srain seems like a fair choice. Although the list can only show 7 applications so I'm not sure what to cut. I could add them in the hope that we show more in the future.

**Comment by mwleeds** [comment]
I agree with only showing free software applications and of course we only want to show applications that work well as a Flatpak (ideally by restricting permissions and using portals rather than just by having loose permissions). But I think popularity/familiarity should also be an important criterion. People probably want to see that they can get the applications they already depend on. So I'm skeptical that taking off GIMP, Inkscape, Atom, VLC, Kdenlive, and Blender is a good idea. Those are all pretty popular. GIMP in particular is both popular and a promoter of Flatpak. Adding LibreOffice seems good. Maybe there's a middle ground where we leave some of those popular FOSS apps in?

**Comment by lionirdeadman** [comment]
As mentioned in the PR, only the 7 first apps are shown so VLC, Kdenlive and Blender were never actually shown.

As for GIMP, Inkscape and Atom, none of them seem to be maintained by their respective developers so I feel showcasing them would be a bit unfair to those who actually put the time to help us although I suppose it could be said that volunteers who help package things also deserve as much credit for their work.

Maybe a future solution could be to show randomly from the list of applications that we think deserve a spotlight as to allow a greater variety of things but I personally don't have the time to even attempt to do it (I have no experience with this codebase) until at minimum next weekend.

**Comment by mwleeds** [comment]
The GIMP flatpak is maintained by upstream

**Comment by lionirdeadman** [comment]
Ah, that's my bad on that then.

What are your thoughts on the randomly shown list? Should we simply make it show more instead?

**Comment by mwleeds** [comment]
I'm not a designer but a randomly chosen list from a curated set seems like a good idea. @allanday might want to give input

**Comment by Seirdy** [comment]
I'm also not certain that "maintained by upstream" is needed; plenty of apps were intentionally designed to be easy to package by third parties.

**Comment by AsciiWolf** [comment]
> A game like 0 A.D. would be nice too. Ideally it'd have something in the "communication" category; I propose Srain (https://flathub.org/apps/details/im.srain.Srain).

I personally think that some more modern IM application would be more appropriate. I think that for example [Fractal](https://flathub.org/apps/details/org.gnome.Fractal) could be a good candidate.
